test/integration/driver.go

```diff
@@ -48,8 +48,8 @@ const (
 	// budget is the maximum number of retries with 1s delays
 	budget = 90
 
-	// Mixer SHA *update manually*
-	mixerSHA = "ea3a8d3e2feb9f06256f92cda5194cc1ea6b599e"
+	// Mixer tag *update manually*
+	mixerTag = "demo"
```

Comment:
Newer images in mixer are not yet tagged with short SHA as per the standard convention. The build tags them with date + build number.  I can use a fix tag like this "2017-03-02-1010".